LayoutTests/http/tests/resourceLoadStatistics/website-data-removal-for-site-navigated-to-with-link-decoration.html

```diff
@@ -5,7 +5,7 @@
     <script src="resources/util.js"></script>
 </head>
 <body onload="setTimeout('runTest()', 0)">
-<div id="description">Check that non-cookie website data gets removed after a navigation with link decoration from a prevalent resource.</div>
```

Comment:
This should be changed to "Check that script-writeable website data …" since we *are* keeping HttpOnly and regular server-set cookies.